src/disco/sbpf/fd_opcode.h

```diff
@@ -0,0 +1,226 @@
+#ifndef HEADER_fd_src_disco_BPF_fd_opcode_h
+#define HEADER_fd_src_disco_BPF_fd_opcode_h
```

Comment:
After we settle on a name for this, should be consistent with name.  (E.g. sbpf is the directory and that's fine but this implies the name is "BPF")